src/lib/numbers.js

```diff
@@ -0,0 +1,57 @@
+const add = (a, b) => {
+  return a + b;
+};
+
+const subtract = (a, b) => {
+  return a - b;
+};
+
+const multiply = (a, b) => {
+  return a * b;
+};
+
+const divide = (a, b) => {
+  return a / b;
+};
+
+const power = (a, b) => {
+  return Math.pow(a, b);
```

Comment:
You have an eslint error on this line. Remember if you hover over the code with red lines below it VSCode shows some info on what the issue is and how to resolve it, in this case it tells you to use the exponentiation operator instead:
```
  return a ** b;
```